src/content/references/en/5-ndvi.md

```diff
@@ -0,0 +1,16 @@
+---
+image: ../../../images/references/ndvi.png
+title: NDVI time-series for field crop monitoring
+---
+
+For an agricultural start-up, we provide a workflow that computes the time series of the mean monthly cloud-free NDVI from Sentinel-2. The time series is automatically extended on a monthly basis. The data comes from a STAC service and does itself not need to be stored. Using APIs, the data can be integrated directly into the customer’s processes.
+
+## Highlights
+
+- Reusable workflows
+- Workflow defined once can be automatically applied to subsequent months to update the product
+- Data retrieval via STAC from the cloud
+
+## Methodik
```

Comment:
should be Methodology

Reply:
[304b434](https://github.com/geo-engine/geoengine.de/pull/6/commits/304b4343dd4c0ea1ab9fdb71988815b9b1b5c145)